src/wallet/subcommands/wallet_cmd.rs

```diff
@@ -252,7 +269,7 @@ pub enum WalletCommands {
     /// Validates whether a given string can be decoded as a well-formed address
     ValidateAddress {
         /// The address to be validated
-        address: String,
+        address: Address,
```

Comment:
Using `Address` here moves the validation to the client side. The point of this command is to validate on the server side. I can't argue for this command's usefulness, but let's do everything client-side or server-side.

Reply:
Gotcha, changed the implementation of `WalletValidateAddress` to match lotus' too